multichain-api-gateway/src/lib.rs

```diff
@@ -0,0 +1,211 @@
+use std::{collections::HashMap, net::TcpListener, str};
+
+use actix_web::{
+    dev::Server,
+    web,
+    web::{Data, Json},
+    App, HttpRequest, HttpResponse, HttpServer, Responder,
+};
+use futures::{stream, StreamExt};
+use reqwest::Client;
+use serde_json::Value;
+use url::Url;
+
+use crate::config::{BlockScoutSettings, Instance, Settings};
+
+mod cli;
+pub mod config;
+
+#[derive(Clone)]
+pub struct APIsEndpoints {
```

Comment:
I believe `ApiEndpoints` would be more consistent with existing guidelines (https://rust-lang.github.io/api-guidelines/naming.html)